build.rs

```diff
@@ -0,0 +1,80 @@
+use std::{
+    path::{Path, PathBuf},
+    process::Command,
+};
+
+use walkdir::WalkDir;
+
+// These two control where this script looks for source and corresponding class files
+const SOURCE_PATH: &str = "java";
+const TARGET_PATH: &str = "target";
+
+// Controls where javac puts generated class files
+const JAVAC_TARGET_PATH: &str = "target/java";
```

Comment:
The `/` separator might be a problem on Windows. Could you use something like [`std::path::MAIN_SEPARATOR`](https://doc.rust-lang.org/std/path/constant.MAIN_SEPARATOR.html)?

Reply:
Yup, I'll just switch to using `join` to avoid the issue entirely.